icu4c/source/test/intltest/dtfmttst.cpp

```diff
@@ -5555,6 +5556,51 @@ void DateFormatTest::TestAdoptCalendarLeak() {
     sdf.adoptCalendar(Calendar::createInstance(status));
 }
 
+void DateFormatTest::Test20741_ABFields() {
+    IcuTestErrorCode status(*this, "Test20741_ABFields");
+
```

Comment:
Add a comment saying what this test is actually testing.  We are testing that the `a` and `B` fields are not duplicated in the field position iterator.